src/components/OrderPanel/BookingTimeForm/FieldDateAndTimeInput.js

```diff
@@ -370,6 +485,9 @@ const onBookingStartDateChange = (props, setCurrentMonth) => value => {
     formApi.change('bookingStartTime', startTime);
     formApi.change('bookingEndDate', { date: endDate });
     formApi.change('bookingEndTime', endTime);
+    if (seatsEnabled) {
+      formApi.change('seats', 1);
+    }
```

Comment:
Mention this behaviour in the description of the PR. It might cause some confusion with anyone who tests things.